common/components/BalanceSidebar/PromoComponents/Simplex.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SimplexLogo from 'assets/images/logo-simplex.png';
+import { NewTabLink } from 'components/ui';
+
+export const Simplex: React.SFC = () => (
+  <NewTabLink className="Promos-promo Promos--simplex" href={`https://buy.mycrypto.com/`}>
+    <div className="Promos-promo-inner">
+      <div className="Promos-promo-text">
+        <h5 key="2">Buy ETH with EUR</h5>
```

Comment:
We'll want this run through `translate`

Reply:
right, forgot about that, I'll add it to the translation files